### PR TITLE
feat: EigenDA <-> OP Plasma deployment guide

### PR DESCRIPTION
**Pull request by epociask**

### CODE REVIEW

File: docs/eigenda/integrations-guides/rollup-guides/op-stack/deployment.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+title: Public Deployment
```
Comment (teddyknox) [inline docs/eigenda/integrations-guides/rollup-guides/op-stack/deployment.md:2]: Deploy Guide ?

File: docs/eigenda/integrations-guides/rollup-guides/op-stack/deployment.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+sidebar_position: 2
+---
+
+# Public Deployment
+
+This guide will show you how to deploy an OP Stack rollup on a public Ethereum
+network, using an EigenDA proxy instance for secure communication with DA.
```
Comment (teddyknox) [inline docs/eigenda/integrations-guides/rollup-guides/op-stack/deployment.md:9]: * Would be good to mention the plasma spec someplace in this sentence.
* "secure communication with the DA network"

File: docs/eigenda/integrations-guides/rollup-guides/op-stack/deployment.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+sidebar_position: 2
+---
+
+# Public Deployment
+
+This guide will show you how to deploy an OP Stack rollup on a public Ethereum
+network, using an EigenDA proxy instance for secure communication with DA. 
+
+## EigenDA Proxy
+EigenDA proxy is a sidecar server which is compatible with the optimism plasma generic server [spec](https://specs.optimism.io/experimental/plasma.html). Instructions for spawning and running the service can be found [here](https://github.com/Layr-Labs/eigenda-proxy). There are two respective modes that can be leveraged when running the integration:
```
Comment (teddyknox) [inline docs/eigenda/integrations-guides/rollup-guides/op-stack/deployment.md:12]: Suggestion:

EigenDA proxy is a sidecar which implements the optimism plasma DA server [spec](https://specs.optimism.io/experimental/plasma.html). Instructions for spawning and running the service can be found [here](https://github.com/Layr-Labs/eigenda-proxy).

Separately we should probably not refer to the memstore mode as anything other than a test mode, and can refer to it much lower in the doc.

**Review by teddyknox — COMMENTED** [review]
Will continue later.

File: docs/eigenda/integrations-guides/rollup-guides/op-stack/deployment.md (hunk 3 above)
Comment (epociask) [reply docs/eigenda/integrations-guides/rollup-guides/op-stack/deployment.md]: [c848a2b](https://github.com/Layr-Labs/eigenlayer-docs/pull/257/commits/c848a2b082478e036c48fe607283eeb598782102)

File: docs/eigenda/integrations-guides/rollup-guides/op-stack/deployment.md (hunk 2 above)
Comment (epociask) [reply docs/eigenda/integrations-guides/rollup-guides/op-stack/deployment.md]: [c848a2b](https://github.com/Layr-Labs/eigenlayer-docs/pull/257/commits/c848a2b082478e036c48fe607283eeb598782102)

File: docs/eigenda/integrations-guides/rollup-guides/op-stack/deployment.md (hunk 1 above)
Comment (epociask) [reply docs/eigenda/integrations-guides/rollup-guides/op-stack/deployment.md:2]: [c848a2b](https://github.com/Layr-Labs/eigenlayer-docs/pull/257/commits/c848a2b082478e036c48fe607283eeb598782102)

**Review by teddyknox — APPROVED** [review]

**Pull request merged**